.gitignore

```diff
@@ -29,3 +29,4 @@
 
 # Ignore master key for decrypting credentials and more.
```

Comment:
Please add a professional README to the project. A professional README should have a project title, project description, and instructions to run the project locally. You may use [this](https://github.com/microverseinc/readme-template) template from Microverse.

Reply:
Done